src/javascript/app/pages/resources/trading_times/trading_times.ui.js

```diff
@@ -48,23 +48,36 @@ const TradingTimesUI = (() => {
         });
         $date.val(localize('Today'));
         if ($(window).width() < 480) {
-            // Create a label to be friendlier
-            const $label = $('label[for=trading-date]');
-            $label.append($('<span/>', { class: 'ux-date foot-note' }));
+            const $input_group = $('#trading-date-container .input-group');
+            const $date_input = $input_group.find('input');
+
+            $input_group.append($('<span/>', { class: 'ux-date foot-note' }));
+
+            const $date_caption = $('span.ux-date');
+
             if (!$date.val()) {
-                $('span.ux-date').text(localize('Today'));
+                $date_caption.text(localize('Today'));
                 $date.val(isoFormattedDate);
                 $date.attr('value', isoFormattedDate);
             }
             $date.change(() => {
                 const diffInDays = moment().diff(moment($date.val()), 'days', true);
                 if (diffInDays < 0 || diffInDays >= 1) {
-                    $('span.ux-date').text('');
+                    $date_caption.text('').hide();
```

Comment:
```suggestion
                    $date_caption.setVisibility(0);
```

Reply:
@mohammad-binary  we can't use `visibility` property due to the fact that it will still remain on top of the datepicker input and it will prevent users from triggering the datepicker popup

Reply:
@akmal-binary  can you also review?